stdlib/datetime.pyi

```diff
@@ -1,14 +1,10 @@
 import sys
 from time import struct_time
-from typing import AnyStr, ClassVar, NamedTuple, Optional, SupportsAbs, Tuple, Type, TypeVar, Union, overload
+from typing import ClassVar, NamedTuple, Optional, SupportsAbs, Tuple, Type, TypeVar, overload
 
 _S = TypeVar("_S")
 
-if sys.version_info >= (3,):
-    _Text = str
-else:
-    _Text = Union[str, unicode]
-
+_Text = str
```

Comment:
Many modules seem to have aliases like this. Perhaps we should clean them up in another PR.